src/context/values.test.ts

```diff
@@ -1,21 +1,7 @@
 import * as assert from "assert";
-import * as sinon from "sinon";
-import { commands } from "vscode";
 import { ContextValues, getContextValue, setContextValue } from "./values";
 
 describe("ContextValue functions", () => {
-  let sandbox: sinon.SinonSandbox;
-  let executeCommandStub: sinon.SinonStub;
-
-  beforeEach(() => {
-    sandbox = sinon.createSandbox();
-    executeCommandStub = sandbox.stub(commands, "executeCommand");
-  });
-
-  afterEach(() => {
-    sandbox.restore();
-  });
```

Comment:
Looks like we lost a test in https://github.com/confluentinc/vscode/pull/714 - did we want to bring it back and keep that sandbox setup?

```ts
  it("setContextValue() should correctly context value and execute the setContext command", async () => {
    const key = ContextValues.<something>;
    const value = ["resource1", "resource2"];

    await setContextValue(key, value);

    assert.ok(executeCommandStub.calledWith("setContext", key, value));
    assert.deepStrictEqual(getContextValue<string[]>(key), value);
  });
```

Reply:
Resurrected, thanks.